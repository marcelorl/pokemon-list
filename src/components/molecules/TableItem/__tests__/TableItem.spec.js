import createTestContext from 'react-cosmos-test/enzyme'

import fixture from '../TableItem.fixture'

const { mount, getWrapper } = createTestContext({ fixture })

beforeEach(mount)

describe('#TableItem', () =>
  it('renders component -> withImage', () => {
    expect(getWrapper()).toMatchSnapshot()
  })
)
