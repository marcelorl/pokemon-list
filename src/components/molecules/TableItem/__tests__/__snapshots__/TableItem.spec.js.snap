// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#TableItem renders component -> withImage 1`] = `
<TableItem
  image="1.png"
  name="With image"
>
  <Col
    className="my-3"
    md="3"
    sm="4"
    tag="div"
    widths={
      Array [
        "xs",
        "sm",
        "md",
        "lg",
        "xl",
      ]
    }
    xs="6"
  >
    <div
      className="my-3 col-6 col-sm-4 col-md-3"
    >
      <Card
        tag="div"
      >
        <div
          className="card"
        >
          <CardImg
            alt="pokemon"
            src="1.png"
            tag="img"
            top={true}
            width="100%"
          >
            <img
              alt="pokemon"
              className="card-img-top"
              src="1.png"
              width="100%"
            />
          </CardImg>
          <CardTitle
            className="text-center"
            tag="h5"
          >
            <h5
              className="text-center card-title"
            >
              With image
            </h5>
          </CardTitle>
        </div>
      </Card>
    </div>
  </Col>
</TableItem>
`;
